apps/andi/lib/andi/input_schemas/datasets/dataset.ex

```diff
@@ -34,6 +36,14 @@ defmodule Andi.InputSchemas.Datasets.Dataset do
     |> cast_assoc(:business, with: &Business.changeset/2)
```

Comment:
do we need to add a ` |> foreign_key_constraint(:owner_id)` here and in the changeset_for_draft function below? I think that would force a dataset to have a user, so idk if we want that or not

Reply:
nvm, just saw your comment in slack